rootx/src/rootx.cxx

```diff
@@ -469,7 +469,7 @@ int main(int argc, char **argv)
    for (i = 1; i < argc; i++) {
       if (!strcmp(argv[i], "-?") || !strncmp(argv[i], "-h", 2) ||
           !strncmp(argv[i], "--help", 6)) {
-         PrintUsage(argv[0]);
+         PrintUsage();
```

Comment:
I am still confused.  Should this PR be closed.   For example, this change is a reversal of:
```
Author: Guilherme Amadio <amadio@cern.ch>
Date:   Fri Sep 21 09:59:00 2018 +0200

    Revert "Final changes to py2help new (#2516)"
    
    This reverts commit dc825fc42a674c2b03c11e1c7e96ca5e261831aa.
```
[ironically that commit is an intentional reversal but this might be a large part of the confusion] ...

If this change are intended, we need to extend the commit log to explain in detail what is the history of this changes. (why a reversal of a reversal, what's the difference, etc.)